### PR TITLE
Print exceptions in MongoHandler

### PR DESCRIPTION
**Pull request by avdv**
This is a reproducer for PR #795.

### CODE REVIEW

File: driver/src/main/scala/core/protocol/MongoHandler.scala [hunk 1]
```diff
@@ -94,6 +94,7 @@ private[reactivemongo] class MongoHandler(
   }
 
   override def exceptionCaught(ctx: ChannelHandlerContext, cause: Throwable) = {
+    println(s"ERROR: $cause")
```
Comment (cchantep) [inline driver/src/main/scala/core/protocol/MongoHandler.scala]: There is a `log` just bellow

Comment (avdv) [reply driver/src/main/scala/core/protocol/MongoHandler.scala]: Yes, and I already told you that *if* one *does* enable the trace log level, that the exceptions would be printed. You, however, stated that "I haven't seen such case locally."

This println statement just makes it obvious that an exception is thrown almost for every single message flowing through the decoder.

I don't know what you want... :confused:

Comment (cchantep) [reply driver/src/main/scala/core/protocol/MongoHandler.scala]: If that's the reproducer, no need to create a PR, just ping me on a branch. That surprised me in this way.

**Comment by avdv** [comment]
Closing, as this was never meant to be merged, I just wanted to trigger a Travis build.

**Pull request closed without merging**